homework/extracts/create_database.sql

```diff
@@ -0,0 +1,14 @@
+
+DROP DATABASE shirley_homework;
+CREATE DATABASE shirley_homework;
+\c shirley_homework;
+
+#code below is not needed because I will make tables in python instead
```

Comment:
Don't commit commented code! Make a separate commit with the code, then another commit with it removed, if you want to be able to return to it.